tools/online_editor/index.html

```diff
@@ -32,7 +32,6 @@
         <label for="style_combo">Style:</label>
         <select id="style_combo">
             <option value="fluent">fluent</option>
-            <option value="ugly">ugly</option>
```

Comment:
well, this is a bit useless now. The combobox should be removed (or commented out in case we want to add another style)

Reply:
This is done now